src/commands/deploy/legacy.ts

```diff
@@ -262,7 +263,9 @@ export default async function main(
   ({ log, error, note, debug, warn } = output);
 
   warn(
-    'You are using an old version of the Now Platform. More: https://zeit.co/docs/v1-upgrade'
+    `You are using an old version of the Now Platform. More: ${
+      link('https://zeit.co/guides/migrate-to-zeit-now/')
```

Comment:
```suggestion
      link('https://zeit.co/guides/migrate-to-zeit-now')
```

Should we drop the trailing slash?